app/routes/circulars._archive._index/route.tsx

```diff
@@ -335,6 +338,22 @@ export default function () {
           <SortSelector form={formId} defaultValue={sort} />
         )}
       </ToolbarButtonGroup>
+      {queryFallback && (
+        <>
+          <Label srOnly htmlFor="query" children={undefined}></Label>
+          <ErrorMessage id="searchFailedHint">
+            "{query}" does not adhere to advanced search syntax. Please refer to
+            the{' '}
+            <Link
+              className="usa-link"
+              to="/docs/circulars/archive#advanced-search"
+            >
+              documentation
+            </Link>{' '}
+            and try again.
+          </ErrorMessage>
+        </>
```

Comment:
```suggestion
          <ErrorMessage>
            "{query}" does not adhere to advanced search syntax. Please refer to
            the{' '}
            <Link
              className="usa-link"
              to="/docs/circulars/archive#advanced-search"
            >
              documentation
            </Link>{' '}
            and try again.
          </ErrorMessage>
```

and then rerun prettier.